docs/redis.md

```diff
@@ -16,6 +16,14 @@ To configure a receiver for your redis metrics, specify the following fields:
 | `collection_interval` | `60s`                     | A [time.Duration](https://pkg.go.dev/time#ParseDuration) value, such as `30s` or `5m`. |
 | `password`            |                           | The password used to connect to the server. |
 
+optional fields are: 
+| Field                 | Default                   | Description |
```

Comment:
What about `insecure` and `insecure_skip_verify`?

Reply:
So,  since docs are now pointing to documentation outside this repo from this [PR](https://github.com/GoogleCloudPlatform/ops-agent/pull/366), how are updates to the documentation going to be added?

Reply:
We are in the middle of automating the documentation related process.

For the moment, the general workflow is:
1. Documentation temporarily goes to GitHub markdown
2. Public doc at https://cloud.google.com/stackdriver/docs/solutions/agents/ops-agent/third-party is derived from the GitHub markdown
3. Once 2 is published, we remove the markdown to avoid having to maintain 2 copies of things.

For this specific case of Redis, options are:
* Leave a section like `Optional fields` in the markdown for now. We'll delete it once the content has been moved to the public side.
* Leave a comment with the documentation change required. We'll take it and add it to the public doc. No need for a follow up cleanup PR.

Reply:
Added comment in doc